libmscore/edit.cpp

```diff
@@ -57,6 +57,8 @@
 #include "textframe.h"
 #include "accidental.h"
 
+#include <QtCore/QtPlugin>
```

Comment:
No it isn't I first thought about a QList, but it is already in the all.h forgot to remove it.